features/step_definitions/networking.rb

```diff
@@ -797,6 +797,21 @@
   raise "the networkType is not #{network_type}" unless @result[:response] == network_type
 end
 
+Given /^the env is using windows container$/ do
+  ensure_admin_tagged
+  _admin = admin
+  @result = _admin.cli_exec(:get, resource: "nodes",show_label:true)
```

Comment:
I think there should be a space after commas  `"nodes", show_label:true)`

Reply:
er, well we could use the `env.nodes` object

```ruby
  is_windows = env.nodes.any? { |n| n.labels["kubernetes.io/os"] == "windows"}
  raise "env doesn't have any windows nodes" unless is_windows
```

Reply:
@rbbratta yea but we intentionally want this because it will dump nodes info with kube version. Most of the time i noticed windows container are one version behind wrt to linux. So it will be helpful in debugging and bugs reporting. I believe env.nodes won't print the nodes ```oc get nodes --show-labels``` info.

Reply:
It will actually print all nodes yaml with each node printing 500 lines so I guess we still favor short and sweet output with only 6-8 lines instead of 3000+ lines.